MULTIPLATFORM.md

```diff
@@ -64,6 +64,15 @@ Compose Multiplatform core libraries can be published to local Maven with the fo
 ```bash
 export COMPOSE_CUSTOM_VERSION=0.0.0-custom-version
 ```
+
+*Note:* instead of using an environment variable, it's possible to use new gradle properties:
+`-Pjetbrains.publication.version.COMPOSE=0.1.0-SNAPSHOT`,
+`-Pjetbrains.publication.version.LIFECYCLE=0.1.0-SNAPSHOT`
```

Comment:
Unfortunately, I noticed a new issue in our CI.

Our `reuploadArtifactsToMavenCentral` task uploads by Compose version. I will modify it later, but before that, artifacts won't be able to be published to Maven Central.